src/arc.ts

```diff
@@ -89,11 +91,91 @@ export function providerHasConfigUi(provider?: any): any | undefined {
   return provider && provider.isTorus;
 }
 
+function getBiconomy(provider: any): any {
+  console.log("getting biconomy");
+  if (biconomy) {
+    return biconomy;
+  }
+  biconomy = new Biconomy(provider, {
+    dappId: "5e961edab93bbb3218f4ca4d",
```

Comment:
this suppose to be a  secret ? if yes lets use env vars for that

Reply:
Sure will put it in env vars.